recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ source:
     - test_Makefile.in.patch
 
 build:
-  number: 10
+  number: 1
   skip: True  # [py==36]
```

Comment:
Please change `py==36` to `win and py==36` and then [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) with the latest `conda-smithy`.